lib/validators/movies/list.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const Joi = require('joi');
+
+module.exports = Joi.object().keys({
+  name: Joi.string().min(1).max(255).optional(),
+  release_year: Joi.alternatives().try(
+    Joi.number().min(1).optional(),
```

Comment:
Does `min(1)` make sense? The technology to create movies didn't exist then. A very ~thorough~ short [Google search](https://headsup.boyslife.org/what-was-the-first-movie-ever-made/) states that 1878 was when the first movie ever was made. Maybe that's a more sensible minimum release year?